cups/raster-interpret.c

```diff
@@ -1302,7 +1303,16 @@ scan_ps(_cups_ps_stack_t *st,		/* I  - Stack */
 	  * Integer with radix...
 	  */
 
-          obj.value.number = strtol(cur + 1, &cur, atoi(start));
+	  base = atoi(start);
+
+	 /*
+	  * strtol() takes 0 or 2 to 32 as base...
+	  */
+
+	  if (base != 0 && (base < 2 || base > 32))
```

Comment:
This should be "base > 36" and don't allow 0 here because that is not valid in PostScript. From the PLRM:

> A radix number takes the form base#number, where base is a decimal integer in the range 2 through 36. The number is then interpreted in this base; it must consist of digits ranging from 0 to base – 1. Digits greater than 9 are represented by the letters A through Z (or a through z). The number is treated as an unsigned integer and is converted to an integer object. This notation is intended for specifying integers in a non-decimal radix, such as binary, octal, or hexadecimal. If the number exceeds the implementations limit for integers, a limitcheck error occurs.